text/0083-func-config-standard.md

```diff
@@ -0,0 +1,66 @@
+- **TEP**: [0](https://github.com/ton-blockchain/TEPs/pull/0)
+- **title**: FunC project configuration file
+- **status**: Draft
+- **type**: Meta
+- **authors**: [Danila Volkov](https://github.com/dvlkv), [Andrey Pfau](https://github.com/andreypfau)
+- **created**: 15.08.2022
+
+# Summary
+
+I propose to make a FunC project config, like project.json, Cargo.toml, etc.
+
+# Motivation
+
+Extensions, compilers and other dev tools need universal project configuration.
+
+# Guide
+
+Extensions can use it for optimal indexing source code, compilers can be configured without bash scripts.  
+Also, here comes more convinient project bootstapping. 
+
+# Specification
+
+I suppose it should be yaml file, named like .func.yml or something else.
+In first implementation it should contain:
+- root source folder
+- entrypoints (contract root files), relative to source folder
+- (optional) library directories
+- (optional) compiler configuration (version, etc.)  
+
+Example file looks like:
+```yml
+сompiler: 
+    version: 0.2.0
+sources: src/
+entrypoints:
+    - contracts/nft.fc
+    - contracts/nft-collection.fc
+libraries:
+    - node_modules/func-*
+```
+
+# Drawbacks
+
+Existing tools with configuration files should be adopted to work correct with this format.
+
+# Rationale and alternatives
+
+- Why is this design the best in the space of possible designs?
+
+In this implementation, configuration file contains only generic information without domain-specific data.
```

Comment:
I think the main idea here is that this structure can be modular and other new third-party tools can simply add own module.